### PR TITLE
Ability to play server from client

### PR DESCRIPTION
**Pull request by andrewgordstewart**
From [this commit](https://github.com/magmo/apps/tree/1acbfa10cf9b3b9165e39c107845604e5db2d081), it should be possible to play a locally hosted node-bot server from the client, with a little local configuration.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -30,6 +31,18 @@ $ curl -X POST -H "Content-Type: application/json" -H "Accept:application/json"
...
+To play against the server from the client, the server's prebuilt contracts will need
+to know the addresses of the deployed contracts on the local ganache network shared between this app and the client app.
+
+In addition to the above development setup, after deploying wallet contracts to your local ganache server, add the wallet artifact's network data to this project's prebuilt contracts:
+```
+chmod +x bin/splice-networks.sh
+PATH_TO_WALLET_CONTRACTS=${PATH_TO_WALLET_PROJECT}/build/contracts yarn splice-networks
+```
+
+You will also need to make sure that the server's address has funds.
```
Comment (kerzhner) [inline readme.md]:
```suggestion
You will also need to make sure that the server's address has funds. You can find the server address in [constants.ts](https://github.com/magmo/node-bot/blob/master/src/constants.ts)
```

File: src/app/services/rpsChannelManager.ts [hunk 2]
```diff
@@ -45,6 +46,9 @@ export async function updateRPSChannel(
     return await openChannel(theirCommitment);
   }
 
+  // Add delay to make it more suspenseful
+  await delay(1000);
```
Comment (kerzhner) [inline src/app/services/rpsChannelManager.ts]: This delay might be cumbersome for any automated or manual integration testing.

**Review by kerzhner — APPROVED** [review]
Looks good, added a couple suggestions

File: src/app/services/rpsChannelManager.ts (hunk 2 above)
Comment (andrewgordstewart) [reply src/app/services/rpsChannelManager.ts]: True.

**Pull request merged**